package.json

```diff
@@ -31,13 +31,13 @@
   },
   "dependencies": {
     "@kaoto/kaoto-ui": "1.0.0",
-    "@kie-tools-core/backend": "^0.27.0",
-    "@kie-tools-core/editor": "^0.27.0",
-    "@kie-tools-core/i18n": "^0.27.0",
-    "@kie-tools-core/monaco-editor": "^0.27.0",
-    "@kie-tools-core/patternfly-base": "^0.27.0",
-    "@kie-tools-core/vscode-extension": "^0.27.0",
-    "@kie-tools-core/workspace": "^0.27.0",
+    "@kie-tools-core/backend": "^0.29.0",
```

Comment:
This upgrade is needed to match the `kaoto-ui` current dependencies, as it is a requirement from `yarn link` to match dependencies.

Reply:
surprised that it is working with kaoto-ui 1.0.0.